packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -144,6 +144,7 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
 
   const microphoneButton = options?.microphoneButton !== false && (
     <MicrophoneButton
+      autoFocus
```

Comment:
Might be worth adding a comment here that this ensures that focus / first tab in is on the microphone button after the `CallComposite` loads.

In general, a nudge to understand that this attribute determines focus for the entire composite.